src/Truncate.js

```diff
@@ -82,16 +82,18 @@ export default class Truncate extends Component {
     // Shim innerText to consistently break lines at <br/> but not at \n
     innerText(node) {
         const div = document.createElement('div');
+        const contentKey = 'innerText' in document.createElement('div') ? 'innerText' : 'textContent';
```

Comment:
I'd prefer to check against

```js
'innerText' in HTMLElement.prototype
```

here, so that we can omit invoking a function and creating an object.